### ISSUE TITLE
Config file given on readme fails because initial_volume is being treated as a string

### ISSUE THREAD

**Issue by TroyNeubauer** [Spotifyd/spotifyd#725]
**Description**
initial_volume is not treated as an integer inside the config file and the demo config file is wrong. 

**To Reproduce**
Build with:cargo build --release --features "pulseaudio_backend"
Copy the config file on the readme to ~/.config/spotifyd/spotifyd.conf - filling in the required options username, device-name etc.

**Expected behavior**
initial_volume in the config file should be allowed to be set without quotes. After doing some digging without looking at the code it looks normalisation_pregain is being parsed as an f32 and thus doesn't require quotes. If this assumption is correct initial_volume should be changed to be parsed as a f32. This will also make the config file on the readme work out of the box because leaving the line `initial_volume = 90` unchanged causes this error. 

**Logs**
Unfortunately it looks like the `<details>` and `<summary>` tags don't work with code blocks
```
/home/troy/Rust/spotifyd/target/release/spotifyd --no-daemon
Loading config from "/home/troy/.config/spotifyd/spotifyd.conf"
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: ErrorInner { kind: Custom, line: Some(64), col: 17, at: Some(1979), message: "invalid type: integer `90`, expected a string", key: ["global", "initial_volume"] } }', src/config.rs:468:67
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
After changing the line `initial_volume = 90` to `initial_volume = "90"` the program runs fine

**Compilation flags** 

- [ ] dbus_mpris
- [ ] dbus_keyring
- [x] alsa_backend
- [ ] portaudio_backend
- [x] pulseaudio_backend
- [ ] rodio_backend

**Versions (please complete the following information):**

- OS: Arch 5.9.10
- Spotifyd: 39106ed8ed270247b1203cc2eed5b05121c90cf7 (latest at the time of writing)
- cargo: cargo 1.48.0 (65cbdd2dc 2020-10-14)

I'm just getting into rust but if you think I'm on the right track I can take a stab at fixing this. 

Here is my complete config file after making the string change. 
```
[global]
# Your Spotify account name.
username = "<snip>"

# Your Spotify account password.
#password = "password"

# A command that gets executed and can be used to
# retrieve your password.
# The command should return the password on stdout.
#
# This is an alternative to the `password` field. Both
# can't be used simultaneously.
password_cmd = "pass spotify"

# If set to true, `spotifyd` tries to look up your
# password in the system's password storage.
#
# This is an alternatie to the `password` field. Both
# can't be used simultaneously.
use_keyring = false

# The audio backend used to play the your music. To get
# a list of possible backends, run `spotifyd --help`.
backend = "alsa"

# The alsa audio device to stream audio to. To get a
# list of valid devices, run `aplay -L`,
device = "alsa_audio_device"  # omit for macOS

# The alsa control device. By default this is the same
# name as the `device` field.
control = "alsa_audio_device"  # omit for macOS

# The alsa mixer used by `spotifyd`.
mixer = "PCM"

# The volume controller. Each one behaves different to
# volume increases. For possible values, run
# `spotifyd --help`.
volume_controller = "alsa"  # use softvol for macOS

# A command that gets executed in your shell after each song changes.
#on_song_change_hook = "command_to_run_on_playback_events"

# The name that gets displayed under the connect tab on
# official clients. Spaces are not allowed!
device_name = "troy-server"

# The audio bitrate. 96, 160 or 320 kbit/s
bitrate = 320

# The directory used to cache audio data. This setting can save
# a lot of bandwidth when activated, as it will avoid re-downloading
# audio files when replaying them.
#
# Note: The file path does not get expanded. Environment variables and
# shell placeholders like $HOME or ~ don't work!
cache_path = "/home/troy/.spotifyd/cache"

# If set to true, audio data does NOT get cached.
no_audio_cache = false

# Volume on startup between 0 and 100
initial_volume = "90"

# If set to true, enables volume normalisation between songs.
volume_normalisation = true

# The normalisation pregain that is applied for each song.
normalisation_pregain = -10

# The port `spotifyd` uses to announce its service over the network.
zeroconf_port = 1234

# The proxy `spotifyd` will use to connect to spotify.
proxy = "http://proxy.example.org:8080"

# The displayed device type in Spotify clients.
# Can be unknown, computer, tablet, smartphone, speaker, tv,
# avr (Audio/Video Receiver), stb (Set-Top Box), and audiodongle.
device_type = "speaker"
```

**Comment by robinvd** [comment]
this might be a duplicate of https://github.com/Spotifyd/spotifyd/issues/698

**Comment by slondr** [comment]
Should be resolved with v0.2.25 (#716)

Let us know if the issue persists
